### ISSUE TITLE
JDK8 Mac build ABORTED : FAILED TO establish the default connection to the WindowServer, _CGSDefaultConnection() is NULL

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#9425]
Failure link
------------

From an internal build `Build_JDK8_x86-64_mac_cm_Nightly/1`.

Optional info
-------------

Failure output (captured from console output)
---------------------------------------------

```
21:46:00  Processing complete
21:46:00  Generating DDR pointer and structure class files
21:46:00  Processing complete
21:46:00  _RegisterApplication(), FAILED TO establish the default connection to the WindowServer, _CGSDefaultConnection() is NULL.
02:12:25  Cancelling nested steps due to timeout
02:12:25  Sending interrupt signal to process
02:12:49  After 20s process did not stop
```

**Comment by pshipton** [comment]
@dnakamura @DanHeidinga

**Comment by dnakamura** [comment]
hmm. No immediate ideas of what it could be, although it is interesting that it seems to hang up right around the time it generates the ddr files, maybe related to #9126, or possibly a coincidence

**Comment by pshipton** [comment]
It's an OpenJ9 jdk8 known problem that occurs when trying to use awt, no idea why it's triggered by the cmake build.

https://github.com/eclipse/openj9/issues/7050#issuecomment-570296489

**Comment by keithc-ca** [comment]
This is not specific to cmake: 3 of the last 13 nightly internal builds failed similarly.

**Comment by keithc-ca** [comment]
On my macOS Catalina 10.15.5 system, using the identical boot JDK (from adoptopenjdk.net) used in the failing builds and a script, I ran the DDR generation tools 200 times: no hang. Perhaps the best solution would be to fix #8841.
